README.md

```diff
@@ -1384,6 +1399,53 @@ out  | evt.ota_end.report      | object     | Sent on upgrade end with upgrade s
    }
 }
 ```
+### Virtual energy service
```

Comment:
This is not a device service, this is an adapter service. As such it should be in the adapter documentation, not this page.

Reply:
Unless your idea is to include this service into any thing that supports virtual energy, which just indicates it is configurable. Once you configure it, one way or another, you will resend the inclusion report?

Reply:
yes, it will send inclusion report. The app needs to know that device support virtual energy to enable panel